### ISSUE TITLE
Magnet link not working. - no ones seeding!

### ISSUE THREAD

**Issue by vizakshat** [deepinsight/insightface#1480]
I am not able to download glint360k dataset even from torrent as no one is seeding. Its been on my torrent client since last month.
can anyone seed and update please!

Hello, we have released glint360k's downloading magnet link,  you can try it.

https://github.com/deepinsight/insightface/blob/master/recognition/partial_fc/README.md#download

_Originally posted by @anxiangsir in https://github.com/deepinsight/insightface/issues/1264#issuecomment-717100093_

**Comment by igorbashko** [comment]
Yes. I loaded part of some files but it stalled currently. Could someone seed the files.

**Comment by maxcrous** [comment]
Same issue here. Haven't had any peers for the past week. It would be great if the authors could re-seed. I have a machine that could seed for the next few months.
